test/calling_test.js

```diff
@@ -1,459 +1,462 @@
-var pretender;
-module('pretender invoking', {
-  setup: function() {
-    pretender = new Pretender();
-  },
-  teardown: function() {
-    if (pretender) {
-      pretender.shutdown();
-    }
-    pretender = null;
-  }
-});
+var describe = QUnit.module;
+var it = QUnit.test;
```

Comment:
I knew this was going to happen eventually after landing the nested modules. I'm glad it worked well for you.